docs/settings.md

```diff
@@ -0,0 +1,61 @@
+# TX
+
++ **Bind**: Enables or disables binding mode.
++ **Band**: Lets you switch between different radio bands _(depending on your build)_.
++ **TX**: >>
+    + **Power**: Cycles between the available transmission power levels for your module.
+    + **Pilot name**: The name of your TX, influences BT name and other things. Can be set through Crossfire's LUA scripts on your radio.
+    + **Input**: Cycles between the available input protocols _(Radio<>Module communication)_. CRSF is suggested for full functionality.
+    + **TX Pin**: Lets you set which pin of your board to use for communicating with the radio. If you followed the [default wiring scheme](tx_module.md#Build) it should be `13`.
++ **Screen**: >>
+    + **Orientation**: Lets you change the orientation of the display to match your board's installation layout.
+    + **Brightness**: Cycles through the available screen brightness levels.
+    + **Auto Off**: Lets you set the timeout for turning off the display. Press a button to turn it on again.
+
++ **Receivers**: >>
+    + Lists bound receivers.
+    + **Receiver #?**: >>
+        + **Name**: The name of the chosen RX _(see [the RX section](#RX))_.
+        + **Address**: Shows the address of this RX _(48 bit number randomly generated at first boot)_.
+        + **Select →**: Choose this RX as the currently active one.
+        + **Delete →**: Delete this RX from the TX's memory. It will have to be rebound.
++ **Other devices**: >>
+    + _// TODO_
++ **Power Test**: _// TODO_
```

Comment:
This just transmits continuously on the centre frequency using the different power levels. It uses the LED to signal each power level so it can be used in modules without screen. First power level makes the led blink once, then twice for the 2nd power level...

However, I plan to rewrite that to use the screen too, so I think it can stay as TODO for now.